### ISSUE TITLE
customize toolbars

### ISSUE THREAD

**Issue by circular17** [bgrabitmap/lazpaint#66]
I work very often visually with mouse and toolbars.
Therefore I would appreciate to have customized toolbar(s) left and top which offer e.g. icons for crop to selection, undo, contrast, rotate.
Pleae add this in the next release
Thank you
Peter

**Comment by lainz** [comment]
Lazarus does something like customizing the toolbars, the interface is very simple, a window with two lists of items, at the left the ones you want to appear in the toolbar (in the order you want them to appear) and at the right all the possible buttons.

**Comment by circular17** [comment]
Yes that would be fine like that. I need for that to make icons for each action.
